src/grading-settings/GradingSettings.jsx

```diff
@@ -153,16 +221,18 @@ const GradingSettings = ({ intl, courseId }) => {
           data-testid="grading-settings-save-alert"
           role="dialog"
           actions={[
-            <Button
-              variant="tertiary"
-              onClick={handleResetGradingCutoffs}
-            >
-              {intl.formatMessage(messages.buttonCancelText)}
-            </Button>,
-            <Button onClick={handleSendGradingSettingsData}>
-              {intl.formatMessage(messages.buttonSaveText)}
-            </Button>,
-          ]}
+            !isQueryPending && (
+              <Button variant="tertiary" onClick={handleResetPageData}>
+                {intl.formatMessage(messages.buttonCancelText)}
+              </Button>
+            ),
+            <StatefulButton
+              key="statefulBtn"
+              onClick={handleSendGradingSettingsData}
+              state={isQueryPending ? RequestStatus.PENDING : 'default'}
```

Comment:
Do we need add disabled props on this page? Or is validation on your page only using a modal window?

Reply:
Thanks
I don't have similar behavior, I hide the select with Save button